src/nodes/constraint_aware_append/constraint_aware_append.c

```diff
@@ -222,6 +222,9 @@ ca_append_begin(CustomScanState *node, EState *estate, int eflags)
 			case T_WorkTableScan:
 			case T_ForeignScan:
 			case T_CustomScan:
+#if PG14_GE
+			case T_TidRangeScan:
+#endif
 			{
 				/*
 				 * If this is a base rel (chunk), check if it can be
```

Comment:
Normal code coverage will not discover if this path is every chosen. Have you investigated and checked that this path is actually reached?

Reply:
No, i could not find out testcase for this case.

Reply:
Same here, i could not find out testcase for this case. However i thought its safe to add the new node.
I think there should be a testcase for atleast IndexScan or BitmapHeapScan which covers the same lines.

Reply:
Could you please investigate if it is possible to add new tests that cover these cases as well?

Reply:
I found it.
Added testcase in compression.sql. This test covers newly added scan node in both the functions.